examples/samg55_xplained_pro/adc-uart/main.cpp

```diff
@@ -19,15 +19,37 @@ using namespace modm::literals;
 modm::IODeviceWrapper<Board::DebugUart, modm::IOBuffer::BlockIfFull> debugDevice;
 modm::IOStream debugStream(debugDevice);
 
+// Rename because the ::Adc typename defined in the CMSIS header conflicts with
+// the name imported from the modm::platform namespace
+using AdcDev = modm::platform::Adc;
+
 int
 main()
 {
+	/** This example reads all 6 ADC channels and print their values to the
+	 * debug UART.
+	 **/
+
 	Board::initialize();
 
-	uint32_t cycle = 0;
+	AdcDev::initialize<Board::SystemClock>();
+	AdcDev::connect<
+		GpioA17::Ad,
+		GpioA18::Ad,
+		GpioA19::Ad,
+		GpioA20::Ad,
+		GpioB0::Ad,
+		GpioB1::Ad>();
+
 	while (true)
 	{
-		modm::delay(1s);
-		debugStream.printf("Cycle: %lu\r\n", cycle);
+		debugStream.printf("ADC Readings: ");
+		for(uint32_t i=0; i<6; i++) 
+		{
+			debugStream.printf("%5d ", modm::platform::Adc::readChannel(i));
```

Comment:
```suggestion
			debugStream.printf("%5d ", AdcDev::readChannel(i));
```
(only for ~my ocd~ consistency)